cmake/modules/CMakeFunctions.cmake

```diff
@@ -1,3 +1,14 @@
+# Add CRT compiler option to avoid link error with Mismatched RuntimeLibrary
+macro(set_msvc_crt_flags name)
```

Comment:
where is this macro from? Do we need to define this macro for out-of-tree build?

Reply:
out-of-tree build doesn't need the macro and is only supported on linux.